cli.js

```diff
@@ -4,8 +4,15 @@ var fs = require('fs');
 var path = require('path');
 var globby = require('globby');
 var meow = require('meow');
+var resolveFrom = require('resolve-from');
 var updateNotifier = require('update-notifier');
 
+try {
+	require(resolveFrom('.', 'babel/register'));
```

Comment:
Actually, should probably do both, in case users use plain `babel`.

```
require(resolveFrom('.', 'babel-core/register') || resolveFrom('.', 'babel/register'));
```